programs/test/benchmark.c

```diff
@@ -1191,7 +1191,7 @@ int main(int argc, char *argv[])
         mbedtls_ecdh_context ecdh_srv, ecdh_cli;
         unsigned char buf_srv[BUFSIZE], buf_cli[BUFSIZE];
         const mbedtls_ecp_curve_info *curve_info;
-        size_t olen;
+        size_t params_len, publen, seclen;
 
         for (curve_info = curve_list;
```

Comment:
Preexisting: I think it would make more sense to iterate over curves, and present the two numbers for each curve consecutively. But for now I prefer to iterate over curves for each handshake type, following what we're doing with ECDSA which has a sign block and a verify block.

Reply:
Agreed, and I'd even go further and group everything (ECDH components + ECDSA ops) for each curve.